### ISSUE TITLE
Simultaneous downloads

### ISSUE THREAD

**Issue by Darkratos** [hydralauncher/hydra#99]
I don't know the extent of libtorrent's capabilities, but would be nice to add several games to download simultaneously (like qBit).

Also, there's no dynamic scraping (at least, not that I've noticed) and it would be cool to do it if possible.

**Comment by FernandoPazCavalcante** [comment]
It would be nice to automatically start the "pending" downloads when the current download finishes though.

**Comment by Darkratos** [comment]
> It would be nice to automatically start the "pending" downloads when the current download finishes though.

Yeah, but I don't see why not just download everything simultaneously. Should be an option to set how many simultaneous downloads the user wants, then queue everything else to start as the current ones finish.

**Comment by FernandoPazCavalcante** [comment]
> > It would be nice to automatically start the "pending" downloads when the current download finishes though.
> 
> Yeah, but I don't see why not just download everything simultaneously. Should be an option to set how many simultaneous downloads the user wants, then queue everything else to start as the current ones finish.

Yes, that would be the best case scenario, for sure.

**Comment by zamitto** [comment]
These changes are obviously possible to do and will probably be done, but they are not so trivial.

We also have some contributors investigating some alternatives to enhance how Hydra manages the torrents downloads. So if someone worked on these changes right now, that might become a wasted work
